analyzers/src/SonarAnalyzer.CSharp/Rules/ComparableInterfaceImplementation.cs

```diff
@@ -143,12 +115,17 @@ private static IEnumerable<string> GetMembersToOverride(IEnumerable<IMethodSymbo
 
             var overridenOperators = methods
                 .Where(m => m.MethodKind == MethodKind.UserDefinedOperator)
-                .Select(m => m.Name);
+                .Select(m => m.ComparisonKind());
 
-            foreach (var op in RequiredOperators.Except(overridenOperators))
+            foreach (var comparisonKind in ComparisonKinds.Except(overridenOperators))
             {
-                yield return OperatorNamesMap[op];
+                yield return comparisonKind.CSharp();
             }
         }
+
+        private static readonly ComparisonKind[] ComparisonKinds =
```

Comment:
Please move this static field to the top of the class to follow our coding style - before or after the other one `ComparableInterfaces`